zookeeper-server/src/main/java/org/apache/zookeeper/jmx/ManagedUtil.java

```diff
@@ -53,7 +53,7 @@ private static boolean isLog4jJmxEnabled() {
     /**
      * Register the log4j JMX mbeans. Set system property
      * "zookeeper.jmx.log4j.disable" to true to disable registration.
-     * @see http://logging.apache.org/log4j/1.2/apidocs/index.html?org/apache/log4j/jmx/package-summary.html
+     * @see <a href="http://logging.apache.org/log4j/1.2/apidocs/index.html?org/apache/log4j/jmx/package-summary.html"/>
```

Comment:
I don't think this will work. This will create a link around an empty region. I was pretty sure a raw URL was fine for the value for `@see`, but if you want to create an explicit link, you need to have something in between `<a ...>` and `</a>`

Reply:
A raw URL still has build warning and I had fixed the empty region.